app/models/sponsorship.rb

```diff
@@ -30,23 +30,12 @@ class Sponsorship < ActiveRecord::Base
 
 private
 
-  def start_date_no_later_than_1st_of_next_month
```

Comment:
at some point it was replaced by the class https://github.com/AgileVentures/osra/blob/e5dc1e6c1f67f5eae8644df3d99a031e9a1b1449/app/models/validators/date_not_beyond_first_of_next_month_validator.rb